src/components/Stream.tsx

```diff
@@ -356,6 +356,8 @@ export const Stream = ({ className = '' }) => {
 
         kclManager.executeAstMock(modifiedAst, true)
       })
+    } else {
+      engineCommandManager.sendSceneCommand(command)
```

Comment:
We were never sending mouse_drag_end events to the engine, but it didn't seem to be a problem 🤷